api/src/main/webapp/WEB-INF/cron_test.yaml

```diff
@@ -0,0 +1,107 @@
+cron:
```

Comment:
@calbach ISTR discussing this with you once. If I want the test environment to have a small change from the default, do I need to past the entire default cron here, or can I include it somehow?